scripts/ingest_GWAS_Catalog.py

```diff
@@ -108,7 +233,7 @@ def parse_associations(association_df: DataFrame) -> DataFrame:
         .withColumn('association_id', f.monotonically_increasing_id())
 
         # Processing variant related columns:
-        #   - Sorting out current rsID field:
+        #   - Sorting out current rsID field: <- why do we need this? rs identifiers should always come from the GnomAD dataset.
```

Comment:
I agree that once you have chromosome, position, ref and alt you can get the rs id from GnomAD. In that case you can remove the lines 247 and 248?

Reply:
So this is complicated... if we join the tables by chromosome and position, there might be multiple variants with different alleles. In those cases it could help the rsID to disambiguate the mapping. This needs to be resolved.